### PR TITLE
Update GetPlayerAnimationFlags.md, added code sample and a warning

### PR DESCRIPTION
**Pull request by m0b-x**
Forgot to add anim flag masks in first commit, did it in the second one.

### CODE REVIEW

**Review by FreddieCrew — COMMENTED** [review]
Looks good to me, except for line 19, which has a small "typo".

> Returns the player animation flags as integer

It's missing the "an" article.

We can take advantage of this pull request and fix this "typo" already.

**Comment by adib-yg** [comment]
I tested the code and it doesn't work
Or possibly the function is broken. It's not even included in omp-stdlib.

<details>
<summary>Screenshots</summary>

sa-mp-170

sa-mp-171

</details>

**Comment by m0b-x** [comment]
Does it work if you add this to your gamemode? 
```c

native GetPlayerAnimFlags(playerid);

```

sa-mp-009

File: docs/scripting/functions/GetPlayerAnimationFlags.md [hunk 1]
```diff
@@ -16,14 +16,48 @@ Get the player animation flags.
...
+#define ANIM_LOCK_X_FLAG   0b0010000000000
+#define ANIM_LOCK_Y_FLAG   0b0001000000000
+#define ANIM_LOOP_FLAG     0b0000100000000
+
+public OnPlayerCommandText(playerid, cmdtext[])
+{
+    if (!strcmp(cmdtext, "/myanimflags"))
+    {
+        new messageString[128];
+        new flags = GetPlayerAnimFlags(playerid);
```
Comment (adib-yg) [inline docs/scripting/functions/GetPlayerAnimationFlags.md:37]:
```suggestion
        new flags = GetPlayerAnimationFlags(playerid);
```

File: docs/scripting/functions/GetPlayerAnimationFlags.md [hunk 2]
```diff
@@ -16,14 +16,48 @@ Get the player animation flags.
...
+
+        new bool:freeze  =   (flags & ANIM_FREEZE_FLAG) != 0 ? true : false;
+        new bool:lockx   =   (flags & ANIM_LOCK_X_FLAG) != 0 ? true : false;
+        new bool:locky   =   (flags & ANIM_LOCK_Y_FLAG) != 0 ? true : false;
+        new bool:loop    =   (flags & ANIM_LOOP_FLAG) != 0 ? true : false;
+
+        format(messageString, sizeof(messageString), "You anim flags are: [freeze:%i] [lockx:%i] [locky:%i] [loop:%i]", freeze, lockx, locky, loop);
```
Comment (adib-yg) [inline docs/scripting/functions/GetPlayerAnimationFlags.md:44]:
```suggestion
        format(messageString, sizeof(messageString), "Your anim flags are: [freeze:%i] [lockx:%i] [locky:%i] [loop:%i]", freeze, lockx, locky, loop);
```

File: docs/scripting/functions/GetPlayerAnimationFlags.md [hunk 3]
```diff
@@ -16,14 +16,48 @@ Get the player animation flags.
...
+        return 1;
+    }
+
+    return 0;
+}
 ```
 
+:::warning
+
+If the player is in a vehicle, all returned animation flags are 0.
```
Comment (adib-yg) [inline docs/scripting/functions/GetPlayerAnimationFlags.md:56]:
```suggestion
If the player state is not on-foot, all returned animation flags are 0.
```

File: docs/scripting/functions/GetPlayerAnimationFlags.md [hunk 4]
```diff
@@ -16,14 +16,48 @@ Get the player animation flags.
...
+
+        format(messageString, sizeof(messageString), "You anim flags are: [freeze:%i] [lockx:%i] [locky:%i] [loop:%i]", freeze, lockx, locky, loop);
+        SendClientMessage(playerid, -1, messageString);
+
+        return 1;
+    }
+
+    return 0;
+}
 ```
```
Comment (adib-yg) [inline docs/scripting/functions/GetPlayerAnimationFlags.md:53]:
```suggestion

## Notes

```

**Review by adib-yg — CHANGES_REQUESTED** [review]

**Pull request merged**